types/ssh2-sftp-client/index.d.ts

```diff
@@ -19,6 +19,7 @@ declare class sftp {
   delete(remoteFilePath: string): Promise<void>;
   rename(remoteSourcePath: string, remoteDestPath: string): Promise<void>;
   end(): Promise<void>;
+  on(event: string|symbol, callback: (...args: any[]) => void): void;
```

Comment:
I think you can omit the `symbol` type for `event` argument

Reply:
I see.
I edit this line. 
thank you.

Reply:
Then for me is good @hiroyky  👍